docs/tutorials/go/audiobook/index.md

```diff
@@ -0,0 +1,1026 @@
+
+1. **Basic understanding of the Temporal Go SDK**:
+   Work through the [Hello World in Go](/getting_started/Go/hello_world_in_Go/) tutorial.
+   This covers the basics of getting a Temporal Go SDK project up and running.
+
+1. **OpenAI API access**:
+   Sign up for your [OpenAI API developer account](https://openai.com/api/) and create a bearer token.
+   You need this token to access OpenAI services for text-to-speech conversion in your project.
+
+Once caught up with prerequisites, you can build out your Go project directory.
+
+:::info CAUTIONS
+
+This is a tutorial project and its implementation is suited for personal and hobbyist use.
+In production, you wouldn't read or write from a single database file, hard drive, or system.
+These aren't durable so you can't develop durable software with them.
+You must be able to rebuild your progress state or store information somewhere more durable.
+If you expand on this project, consider a API-based Cloud storage solution.
+
+:::
+
+## Create your Go project structure
+
+1. Set up your project directory:
+   Set up your source code folder hierarchy by issuing this directory creation command:
+
+   ```sh
+   mkdir -p TTSWorker/worker
+   ```
+
+   Your directory structure should now look like this:
+
+   ```text
+   TTSWorker
+   └── worker
+   ```
+   
+2. Move to the root directory:
+   
+  ```sh
+   cd TTSWorker
+   ```
+
+3. Create your Makefile in the `TTSWorker/` project root directory.
+   Remember that Makefile uses tabs not spaces for indentation when creating this file:
+
+   ```makefile
+   # Go parameters
+   GOCMD = go
+   GOBUILD = $(GOCMD) build
+   GOCLEAN = $(GOCMD) clean
+   GORUN = $(GOCMD) run
+   GOGET = $(GOCMD) get
+   GOINSTALL = $(GOCMD) install
+   GOPATH ?= $(shell go env GOPATH)
+   BINARY_DIR = bin
+   BINARY_NAME_WORKER = $(BINARY_DIR)/worker
+
+   all: build
+
+   clean:
+       $(GOCLEAN)
+       rm -rf $(BINARY_DIR)
+
+   build: $(BINARY_DIR) build-worker
+
+   $(BINARY_DIR):
+       mkdir -p $(BINARY_DIR)
+
+   build-worker:
+       cd worker && $(GOBUILD) -o ../$(BINARY_NAME_WORKER)
+
+   run:
+       ./$(BINARY_NAME_WORKER)
+
+   .PHONY: all clean build build-worker clean-deps fetch-deps worker
+   ```
+
+4. Create `go.mod` by entering:
+
+   ```
+   go mod init audiobook/app
+   ```
+   
+   Edit your new `go.mod` file to add two requirements:
+
+   ```
+   module audiobook/app
+
+   go 1.22.2
+
+   require (
+       go.temporal.io/sdk v1.17.0
+       github.com/stretchr/testify v1.8.0
+   )
+   ```
+   
+   Your dependencies include:
+   
+   * **go.temporal.io/sdk v1.17.0**: Temporal's open source failure mitigation Go SDK.
+   * **github.com/stretchr/testify v1.8.0**: Write tests with assertions and mocks.
+
+5. Establish your bearer token environment variable
+   Your application uses the environment variable to authenticate with the OpenAI service.
+   It needs to live in the same shell as  your running app.
```

Comment:
"live in the same shell" is potentially very unclear to junior devs who don't think much about env variables, and I definitely don't think we need to document tcsh -- almost every beginner tutorial like this I've seen defaults to bash syntax. I would simplify all of this and just focus on prepending the `export` syntax later on when it's actually used.

Reply:
I'll change it to bash, and I'm going to use terminal session.